docs/community/com-cluster/com-cluster-manage.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Community cluster
+layout: default
+parent: Community
+has_children: true
+---
+
+# How do I manage a FeatureBase Community cluster?
+
+A FeatureBase Cluster is a distributed system that is maintained by `etcd` which runs on each node.
+
+The system requires at least three nodes to maintain data consistency and fault tolerances. <!--from https://stackoverflowteams.com/c/molecula/questions/179-->
+
+## Before you begin
+{% include /com-install/com-install-before-begin.md %}
+
+## How do I setup a cluster?
+
+* [Learn how to setup a FeatureBase cluster](/docs/community/com-cluster/com-cluster-setup)
+
+## How do I resize a cluster?
```

Comment:
The old doc had info about making sure to stop ingest, point to new cluster, and things like that. Just making sure you think we should cut that info out?

Reply:
Yes on this -- I need to determine how best to talk about this because there's 2 ways of setting up FB to run:
* ./featurebase server
* via systemctl commands

Right now there's a single page for starting/connecting to Fb via /opt and another for the systemctl setup and the commands. I'm going to revisit these once the old content is fixed up.

Reply:
The other thing is that there's no information on HOW to stop ingest and point to a cluster anywhere in docs. So thats a big gap.

Reply:
All the information about that is actually referring to a backup process (which is already documented). I've double-checked the original file in the old /documentation/community/community-database/_posts/2022-04-01-resize-community-database.md

Reply:
What I don't know is if the restore gets done on *one* or *all* the cluster nodes, then restored on each in turn?

Reply:
Also, it may be worthwhile creating cluster-centric copies of the backup/restore docs to put emphasis on the cluster specifics. Right now the backup/restore refer to these in content